src/pages/futurehack-2021.js

```diff
@@ -0,0 +1,462 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import cx from 'classnames';
+import PageLayout from '../components/PageLayout';
+import { Button, Icon } from '@newrelic/gatsby-theme-newrelic';
+import DevSiteSeo from '../components/DevSiteSeo';
+import { graphql, useStaticQuery, Link } from 'gatsby';
+import styles from './futurehack.module.scss';
```

Comment:
we've also changed this import syntax and should now use `import * as styles from './futurehack.module.scss';`